pkg/probe/http/http.go

```diff
@@ -22,20 +22,67 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/url"
+	"path/filepath"
 	"time"
 
 	utilnet "k8s.io/apimachinery/pkg/util/net"
 	"k8s.io/kubernetes/pkg/probe"
 	"k8s.io/kubernetes/pkg/version"
 
+	"crypto/x509"
 	"github.com/golang/glog"
 )
 
-func New() HTTPProber {
+const (
+	defaultKubeletClientCertificateFile = "kubelet-client.crt"
+	defaultKubeletClientKeyFile         = "kubelet-client.key"
+	defaultKubeletCACertificateFile     = "kubelet-ca.crt"
```

Comment:
hard-coded files like this aren't appropriate... I'd expect these to come from kubelet config

Reply:
also, it's unclear how this connects to the client credential management the kubelet has (client cert rotation, initialized from the `--kubeconfig` arg, etc)